src/symbolOutline.ts

```diff
@@ -155,10 +150,12 @@ export class SymbolOutlineProvider implements TreeDataProvider<SymbolNode> {
         const { kind } = node.symbol;
         let treeItem = new TreeItem(node.symbol.name);
         treeItem.collapsibleState = node.children.length ? TreeItemCollapsibleState.Collapsed : TreeItemCollapsibleState.None;
+        let range = node.symbol.location.range;
+        range = range.with(range.start, range.start);
```

Comment:
Ah, I added this commit to my branch not realizing it would update the pull request.  I'll revert.

The change itself is something I prefer but maybe isn't for everyone.  Right now if you click on a class that takes more than one screen it selects the entire class, and scrolls to the middle of the class.  With that change it doesn't select the entire class but instead just goes to the beginning of the class which I prefer.